resources/dw/resource_hive_vw_acc_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2024 Cloudera. All Rights Reserved.
+//
+// This file is licensed under the Apache License Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, either express or implied. Refer to the License for the specific
+// permissions and limitations governing your use of the file.
+
+package dw_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/cloudera/terraform-provider-cdp/cdp-sdk-go/gen/dw/client/operations"
+	"github.com/cloudera/terraform-provider-cdp/cdp-sdk-go/gen/dw/models"
+	"github.com/cloudera/terraform-provider-cdp/cdpacctest"
+	"github.com/cloudera/terraform-provider-cdp/utils"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+)
+
+type hiveTestParameters struct {
+	Name              string
+	ClusterID         string
+	DatabaseCatalogID string
+}
+
+var (
+	preCheckOnce sync.Once
```

Comment:
if you add multiple acceptance tests for the same package the `var` name will collide. What's the reason for doing this check only once? Can we simplify here by doing the checks always? The env vars rarely change throughout the run...

Reply:
yes we can simplify

Reply:
removed this